### PR TITLE
[Azure Pipelines] Enable TBPL logging for infrastructure/ tests

### PR DESCRIPTION
**Pull request by foolip**
This matches tools/ci/ci_wptrunner_infrastructure.sh.

### CODE REVIEW

**Comment by foolip** [comment]
I did this PR mostly to debug https://github.com/web-platform-tests/wpt/pull/13831, I don't really like the output format. @jgraham, do you think we should change tools/ci/ci_wptrunner_infrastructure.sh instead?

**Comment by jgraham** [comment]
I mean I don't love the tbpl format either but it's designed for CI use cases and is the one supported by the reftest analyzer so I think it makes more sense to land this than change the other thing.

File: .azure-pipelines.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ jobs:
 
   - script: |
       echo "Test jobs:"
-      ./wpt test-jobs | while read job; do
+      ./wpt test-jobs --all | while read job; do
```
Comment (jgraham) [inline .azure-pipelines.yml]: Need to revert this change, right?

**Review by jgraham — APPROVED** [review]
Assuming that you're reverting the `--all` change.

**Comment by foolip** [comment]
> I mean I don't love the tbpl format either but it's designed for CI use cases and is the one supported by the reftest analyzer so I think it makes more sense to land this than change the other thing.

I tried downloading the log from this PR and putting it into the reftest-analyzer, but it doesn't work because there's a bunch of extra stuff, timestamps and a header at the top. I tried to trim it manually but seems to have failed.

I could instead log to a file and put that in artifacts for debugging. Would you like me to do that, or wait until there's a problem to debug?

**Comment by jgraham** [comment]
Wait until there's a problem to debug.
